mono/eglib/goutput.c

```diff
@@ -139,6 +139,7 @@ g_assertion_message (const gchar *format, ...)
 {
 	va_list args;
 
+	va_start (args, format);
```

Comment:
We are not sure this is portable and  it is not efficient -- formats twice.
The master fix avoids both  problems.

Reply:
@jaykrell merged onto 2018-02 is code that moves the va_start below the other function rather than making two. I had misunderstood your feedback before. I think that a fix/undo for that should be merged.